### ISSUE TITLE
Appears to be Crashing when Scanning files downloaded by Tubesync

### ISSUE THREAD

**Issue by uhdoy** [ZeroQI/Absolute-Series-Scanner#446]
## Platform

Operating system and version:Ubuntu 22.04 LTS
Plex version: 1.32.5.7349-8f4248874

## Expected Behavior

Scan and name files
## Current Behavior

Appears to crash part way through scanning, does not rename. I don't believe the work gets handed off to the Agent.

## Steps to Reproduce

1. Add New Library
2. Select TV
3. Select Folder of downloaded youtube videos
4. Select Scanner as Absolute Series, Agent as YoutubeSeries  (screenshots of library configuration in Additional info section)
5. Save Library

## Additional information

I've tried/retried this many times, the behavior isn't always the same. I've also received the following error, however my API key is correct.  I apologize if I missed any steps, I have read and re-read the post so I'm hopeful I didn't miss a step.  Thank you.

When scanning in the files, for some reason it's lumping multiple files together and has a wonky title. 

As an example - this "show" should only have 4 episodes, and it has 19.

Example:https://imgur.com/a/D2USjAL
I'll didn't save all the links but I think I've tried about everything I've seen. My current setup:

- Plex
- TubeSync
- Scanner: Absolute Series Scanner (added API key to preferences xml file)
- Agent: YoutubeSeries w/ API Key

Screenshots of the various configurations:

- TubeSync: https://imgur.com/a/nQ17Uqq
- Plex: https://imgur.com/a/sUABRVg
- Scanner & Agent:https://imgur.com/a/IrP18jD
- com.plexapp.agents.youtube.xml: https://pastebin.com/yNhGMMRA

Example Playlist: https://www.youtube.com/playlist?list=PLUR-PCZCUv7ROzIyo8VdHkcFrS0qmbJtz
Example NFO File created when downloaded: https://pastebin.com/DwyMpUnL
Plex Media Scanner Log: https://pastebin.com/fySFcm13

**ASS Logs**

- _root_.scanner.log: https://pastebin.com/M9dMScYw
- DanTDM.scanner.log: https://pastebin.com/62PNaV9u

**Comment by uhdoy** [comment]
For the heck of it, I executed a Plex Dance and retried Scanning:

post dance root scanner log: https://pastebin.com/f5j6MrPW
post dance DanTDM scanner log: https://pastebin.com/isMye68X

**Comment by uhdoy** [comment]
This can be chalked up to user error.  I had failed to activate the Youtube API, I had only created the key. I am sure this is user error.

To fix:
Go to "ENABLE APIS AND SERVICES"
![image](https://github.com/ZeroQI/Absolute-Series-Scanner/assets/140649492/e2c419fb-5d40-466a-ad2f-8d9bf0d19759)

Search for Youtube data api v3
![image](https://github.com/ZeroQI/Absolute-Series-Scanner/assets/140649492/7865a75f-40ee-4a4d-916f-57bb1fb63374)

CLick the link and choose enable
